jjb/egeria/egeria.yaml

```diff
@@ -10,6 +10,9 @@
     stream:
       - 'master':
           branch: master
+    java-version:
```

Comment:
Is this supported? 2 JDKs?

Reply:
> I am not sure the commit text matches the file changes since we are not creating new jobs.
> 
> Can we please fix that?

By this change there will be a new jobs generated - odpi-egeria-maven-verify-master-mvn35-openjdk11, egeria-connector-ibm-igc-maven-verify-master-mvn35-openjdk11. 
Do I need to change the commit message to generating.. ??

Reply:
> Is this supported? 2 JDKs?

Yes, With the new Image it's supported. Tested them in sandbox.